libs/sdk-ui-ext/src/internal/components/configurationControls/colors/colorDropdown/tests/ColorPalette.test.tsx

```diff
@@ -15,55 +15,52 @@ const defaultProps: IColorPaletteProps = {
 
 function createComponent(customProps: Partial<IColorPaletteProps> = {}) {
     const props: IColorPaletteProps = { ...cloneDeep(defaultProps), ...customProps };
-    return mount<IColorPaletteProps>(<ColorPalette {...props} />);
+    return {
+        user: userEvent.setup(),
+        ...render(<ColorPalette {...props} />),
+    };
 }
 
 describe("ColorPalette", () => {
-    it("should render ColorPalette control", () => {
-        const wrapper = createComponent();
-        expect(wrapper.find(ColorPalette).length).toBe(1);
-    });
+    afterEach(cleanup);
 
     it("should render small ColorPalette control", () => {
-        const wrapper = createComponent();
-        expect(wrapper.find("div").first().hasClass("gd-color-drop-down-list")).toBeTruthy();
-        expect(wrapper.find("div").first().hasClass("gd-color-drop-down-list-large")).toBeFalsy();
+        const { getAllByLabelText } = createComponent();
+        expect(getAllByLabelText(/rgb*/i).length).toBe(5);
     });
 
     it("should render large ColorPalette control", () => {
-        const wrapper = createComponent({ colorPalette: getLargePalette() });
-        expect(wrapper.find("div").first().hasClass("gd-color-drop-down-list")).toBeFalsy();
-        expect(wrapper.find("div").first().hasClass("gd-color-drop-down-list-large")).toBeTruthy();
+        const { getAllByLabelText } = createComponent({ colorPalette: getLargePalette() });
+        expect(getAllByLabelText(/rgb*/i).length).toBe(20);
     });
 
-    it("should render 5 ColorPaletteItem controls", () => {
-        const wrapper = createComponent();
-        expect(wrapper.find(ColorPaletteItem).length).toBe(5);
+    it("should render 5 ColorPaletteItem controls one has to be selected", () => {
```

Comment:
This no longer tests if there are 5 items. Either test that fact or change the name of the test.

Reply:
I have updated the test